### PR TITLE
Travis CI with lint checking added

### PR DESCRIPTION
**Pull request by lalitmee**

### CODE REVIEW

**Comment by lalitmee** [comment]
Hey @aashutoshrathi, why all these previous commits are coming in this PR. Since we have merged all the previous commits. :thinking:

**Comment by aashutoshrathi** [comment]
Your master is not up to date.
Can you please do that

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - '6'
+
+script:
+  - npm run lint
```
Comment (aashutoshrathi) [inline .travis.yml]: We need to do, `npm i` too

Comment (aashutoshrathi) [reply .travis.yml]: Also, npm run lint is not working

**Comment by lalitmee** [comment]
I have synced my master, still, these commits are there.

File: .travis.yml (hunk 1 above)
Comment (lalitmee) [reply .travis.yml]: Oh yeah

**Comment by aashutoshrathi** [comment]
:sweat_smile: 

screenshot from 2018-12-26 12-25-22

**Comment by aashutoshrathi** [comment]
In package.json make line 8 as `"lint": "./node_modules/.bin/eslint . --fix"` and it will work.

File: .travis.yml (hunk 1 above)
Comment (lalitmee) [reply .travis.yml]: Then I think we need this in our scripts

```json
"scripts": {
    "lint": "./node_modules/.bin/eslint . ./**/*.js"
  }
```

**Comment by aashutoshrathi** [comment]
One more thing.
Add these two lines on the top of `download.js`
```js
/* eslint no-undef: "error" */
/* eslint-env browser */
```

**Comment by aashutoshrathi** [comment]
I would recommend you to create a new fork since it's all messed up with commit history.

**Comment by lalitmee** [comment]
Okk i will create a new fork.

**Pull request merged**